apps/sel4bench/src/processing.h

```diff
@@ -17,6 +17,10 @@
  */
 result_t process_result(size_t n, ccnt_t array[n], result_desc_t desc);
 
+/*  For Early Processing configuration  */
+result_t process_result_early_proc(ccnt_t num, ccnt_t min, ccnt_t max,
```

Comment:
what is in `array[num]`? Docs would help :)

Reply:
Done